low-level/files/opt/seagate/sspl/setup/sspl_test.py

```diff
@@ -40,7 +41,7 @@ def __init__(self, args: list):
         self.args = args
         self.name = "sspl_test"
         self.plan = "self"
```

Comment:
Change self.plan = "sanity" as we have renamed "self" plan to "sanity".

Reply:
addressed 👍